infrastructure/src/main/java/org/corfudb/infrastructure/RemoteMonitoringService.java

```diff
@@ -496,11 +504,9 @@ private DetectorTask detectHealing(PollReport pollReport, Layout layout) {
      * @param pollReport Poll report obtained from failure detection policy.
      * @return boolean result if failure was handled. False if there is no failure
      */
-    private DetectorTask detectFailure(PollReport pollReport) {
+    private DetectorTask detectFailure(PollReport pollReport, Layout layout) {
```

Comment:
There are 2 methods with the same name. Please change the names to a more intuitive name.
For example, you can name the other one as handleFailure as it's sending a message to handle the detected failures.

Reply:
Last time you asked exactly a vise-versa thing) 
if you remember you asked to rename that method from `handleFailure` to `detectFailure` because we have `handleFailure`:
```
return getCorfuRuntime()
                .getLayoutView()
                .getRuntimeLayout(layout)
                .getManagementClient(serverContext.getLocalEndpoint())
                .handleFailure(pollReport.getPollEpoch(), failedNodes)
                .thenApply(DetectorTask::fromBool);
```